scripts/linters/js_ts_linter.py

```diff
@@ -284,6 +284,54 @@ def _check_any_type(self):
 
         return [summary_message]
 
+    def _check_http_requests(self):
+        """Checks if the http requests are made only by
+        backend-api.service.ts.
+        """
+
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting HTTP requests check')
+            python_utils.PRINT('----------------------------------------')
+
+        http_client_pattern = r':\n? *HttpClient'
+
+        excluded_files = [
+            'core/templates/services/request-interceptor.service.spec.ts'
+        ]
+
+        with linter_utils.redirect_stdout(sys.stdout):
+            failed = False
+
+            for file_path in self.all_filepaths:
+                if file_path in excluded_files:
+                    continue
+
+                file_content = FILE_CACHE.read(file_path)
+
+                if re.findall(http_client_pattern, file_content):
+                    if not file_path.endswith('backend-api.service.ts'):
```

Comment:
Move line 312 to 310 and change it to 
`if file_path.endswith('backend-api.service.ts'):`
`continue`

Reply:
done